packages/eslint-config/rules/style.js

```diff
@@ -50,8 +50,12 @@ module.exports = {
 
     'arrow-spacing': 2,
 
+    'brace-style': 2,
+
     'comma-dangle': [`error`, `always-multiline`],
 
+    'comma-spacing': 2,
```

Comment:
You should enable `@typescript-eslint/comma-spacing`, just like I did in my PR. :P

Reply:
Yup, noticed, going over more rules atm